src/components/Button.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Button(props) {
+  const { name } = props;
+  return (
+    <>
+      Button name:
+      {name}
+    </>
```

Comment:
- This should be an actual HTML button component.
```react
<>
<button>...</button>
</>
```

Reply:
Done